src/views/home/components/list/list.view.tsx

```diff
@@ -59,33 +60,44 @@ const List: FC<ListProps> = ({ placeholder, list, onClose }) => {
     onClose();
   };
 
+  useEffect(() => {
+    if (inputRef && inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, []);
+
   return (
     <Portal>
       <div className={classes.background} onClick={onOutsideClick}>
         <Card className={classes.card}>
           <div className={classes.search}>
             <SearchIcon />
-            <input className={classes.input} placeholder={placeholder} onChange={onChange} />
+            <input
+              ref={inputRef}
```

Comment:
We could remove `Search Network input`  while we've just two supported networks

Reply:
I agree, it was on my plan. I've commented it temporarily.